kuma/wiki/models.py

```diff
@@ -1584,7 +1598,10 @@ class DocumentZone(models.Model):
     attributes inherited by the topic hierarchy beneath it.
     """
     document = models.OneToOneField(Document, related_name='zone')
-    styles = models.TextField(null=True, blank=True)
+    css_filename = models.CharField(
+        max_length=100, null=True, blank=True,
```

Comment:
Yeah, I think that's better, if only so in the future we don't end up with a mixture both NULL's and empty strings in that column.